core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -819,13 +818,14 @@ bindings(Routing) ->
                              }=Bind, Acc) ->
                       case Binding =:= Routing
                           orelse matches(BParts, RoutingParts)
-                          orelse matches(RoutingParts, BParts)
```

Comment:
This line was introduced by https://github.com/2600hz/kazoo/pull/2395/files#diff-361a0ac2e37e647a171e2f5518a79ee7 And as per @jamesaimonetti bindings should not "look both ways"...